frontend/src/hocs/withExpandable/withExpandable.tsx

```diff
@@ -89,9 +88,16 @@ function withExpandable(
                 borderBottomStyle: "none",
                 borderRightStyle: "none",
                 borderLeftStyle: "none",
+                overflow: "visible",
```

Comment:
nit: can you move that change up above `paddingLeft` so it's not shown as an change here.